### DISCUSSION TITLE
C dependency geos: Plan for refresh from original repo

### DISCUSSION THREAD

**Discussion by prashantkhoje** [cockroachdb/cockroach#81237] [Q&A] [answered]
Cockroachdb has forked package geos to be used with it.
The forked repo is ~2 year old.
Is there plan to refresh it from [original repo](https://github.com/libgeos/geos)?
On ppc64le, i found that forked repo has 12 failures:
>          11 - unit-algorithm-LocatePointInRing (Failed)
>          14 - unit-algorithm-OrientationIndexFailure (Failed)
>          17 - unit-algorithm-RobustLineIntersector (Failed)
>          82 - unit-capi-GEOSVoronoiDiagram (Failed)
>         116 - unit-geom-Triangle (Failed)
>         129 - unit-linearref-LengthIndexedLine (Failed)
>         130 - unit-math-DD (Failed)
>         142 - unit-operation-distance-DistanceOp (Failed)
>         177 - general-TestCentroid (Failed)
>         191 - general-TestFunctionPLPrec (Failed)
>         222 - issue-issue-geos-350 (Failed)
>         269 - robust-TestRobustRelate (Failed)

Whereas, original geos has only 1 failure:
>        243 - unit-shape-fractal-HilbertEncoder (Failed)

I understand ppc64le is not supported by cockroachdb.

Thank you,
Prashant

**Comment by prashantkhoje** [comment]
@otan, Would you be able to comment on this?

Thank you,
Prashant

**Comment by otan** [comment, marked as answer]
Hey @prashantkhoje, no immediate plans, but I'll try make time for it later this release. It will be at least 6 months until we refresh, however.

**Comment by otan** [comment]
Moved to #81290
